ownCloud/Client/Actions/EditDocumentViewController.swift

```diff
@@ -28,6 +28,7 @@ class EditDocumentViewController: QLPreviewController, Themeable {
 	var savingMode: QLPreviewItemEditingMode?
 	var itemTracker: OCCoreItemTracking?
 	var modifiedContentsURL: URL?
+	var dismissedViewWithoutSaving: Bool = false
```

Comment:
@hosy may be you could use more descriptive name? E.g. `editingModeLeftWithUnsavedContent`?

Reply:
It does not reflect the name in my opinion, because it is important, that view was dismissed without unsaved content.

Reply:
your suggested name does not reflect for what the variable is, because it is important, that the view was dismissed. What is with `dismissedViewWithUnsavedContent`?

Reply:
@hosy then, I am also fine if you leave it as is...